php/admin/class-settings.php

```diff
@@ -122,7 +122,7 @@ public function prepare_notice( $notice ) {
 	 * Show any admin notices after saving the settings.
 	 */
 	public function show_notices() {
-		$notices = get_option( 'block_lab_notices', array() );
+		$notices = get_option( 'block_lab_notices', [] );
```

Comment:
I realized that by changing `phpcs.xml`, doing `npm run lint:php:fix` runs `phpcbf` can convert all of the `array()` syntax to `[]`.

This produced a huge diff, and will of course need regression testing.

But I think it's important for Block Lab to be modern. The `[]` syntax [started in PHP 5.4](https://www.php.net/manual/en/language.types.array.php), and we only support PHP 5.6+.